src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -9,6 +9,15 @@ public class Driver {
     private Long carId;
     private String login;
     private String password;
+    private String role;
```

Comment:
@EliosPride why string? you must use enum

Reply:
@Garret-vin +